lib/pluginmanager/clean.rb

```diff
@@ -0,0 +1,23 @@
+require "pluginmanager/command"
+
+class LogStash::PluginManager::Clean < LogStash::PluginManager::Command
+
+  def execute
+    signal_error("File #{LogStash::Environment::GEMFILE_PATH} does not exist or is not writable, aborting") unless File.writable?(LogStash::Environment::GEMFILE_PATH)
+
+    ##
+    # Need to setup the bundler status to enable uninstall of plugins
+    # installed as local_gems, otherwise gem:specification is not
+    # finding the plugins
+    ##
+    LogStash::Bundler.setup!({:without => [:build, :development]})
```

Comment:
this should not be required, the setup is already done in the `invoke!` method. I am guessing you took this from the `uninstall` command, in which case the `setup!` call is required for the `LogStash::PluginManager.installed_plugin?` call.